prometheus-promu/promu.spec

```diff
@@ -0,0 +1,61 @@
+Name:           prometheus-promu
```

Comment:
Do we need this package?

Reply:
yes, it is used to build to prometheus itself.